src/sentry/api/endpoints/release_threshold.py

```diff
@@ -0,0 +1,66 @@
+from django.http import HttpResponse
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.rest_framework.environment import EnvironmentField
+from sentry.api.serializers.rest_framework.project import ProjectField
+from sentry.models import Project
+from sentry.models.release_threshold.constants import (
+    THRESHOLD_TYPE_STR_TO_INT,
+    TRIGGER_TYPE_STRING_TO_INT,
+)
+from sentry.models.release_threshold.releasethreshold import (
```

Comment:
Ah - if you're adding the folder to the models, you can just leave this one as the `__init__.py` file maybe?

Reply:
There aren't currently many model folders though (🤷 )

So maybe we move the constants to the api/endpoints/release_thresholds?

Reply:
Synced in person re. circular imports